client/client.go

```diff
@@ -214,6 +188,36 @@ func (c *LogClient) UpdateRoot(ctx context.Context) error {
 	return nil
 }
 
+// WaitForInclusion blocks until the requested data has been verified with an inclusion proof.
+func (c *LogClient) WaitForInclusion(ctx context.Context, data []byte) error {
+	leaf := c.logVerifier.buildLeaf(data)
+
+	// Fetch the current Root to improve our chances at a valid inclusion proof.
+	if err := c.UpdateRoot(ctx); err != nil {
+		return err
+	}
+	if c.root.TreeSize == 0 {
+		return fmt.Errorf("Tree is empty, TreeSize = 0")
```

Comment:
It's an error to request an inclusion proof with TreeSize == 0. 
I'll fix this to wait for a root update.